src/ucp/core/ucp_ep.inl

```diff
@@ -32,10 +32,10 @@ static inline ucp_lane_index_t ucp_ep_get_wireup_msg_lane(ucp_ep_h ep)
     return (lane == UCP_NULL_LANE) ? ucp_ep_get_am_lane(ep) : lane;
 }
 
-static inline ucp_lane_index_t ucp_ep_get_rndv_get_lane(ucp_ep_h ep)
+static inline ucp_lane_index_t ucp_ep_get_rndv_get_lane(ucp_ep_h ep, int idx)
```

Comment:
Here and below, should `idx` be `ucp_lane_index_t ` type?

Reply:
technically lane_index may be used, but this is index in table of references to lane_indexes.
if you don't like 'int' as input parameter we may add new datatype

Reply:
Can this table size be > num lanes (rails)? if this is equivalent things, i would prefer `ucp_lane_index_t`